client/components/sites-add-new-site/content/index.tsx

```diff
@@ -118,18 +113,16 @@ export const Content = () => {
 							comment: 'percentage like 55% off',
 						},
 					} ) }
-					description={ preventWidows(
-						translate(
-							'Save up to %(percentage)s on annual plans and get a free custom domain for a year. Your next site is just a step away.',
-							{
-								args: {
-									percentage: formatNumber( 0.55, {
-										numberFormatOptions: { style: 'percent' },
-									} ),
-									comment: 'percentage like 55% off',
-								},
-							}
-						)
```

Comment:
Add new site popover:

<img width="970" alt="Screenshot 2025-05-12 at 2 52 27 pm" src="https://github.com/user-attachments/assets/91f1ceca-8a75-4e26-b508-c46d3fbe32d8" />

Though the text is wrapped in `div`s, heading and description already have custom styles applied.